ddl/ddl_api.go

```diff
@@ -347,6 +359,26 @@ func columnDefToCol(ctx sessionctx.Context, offset int, colDef *ast.ColumnDef) (
 				// TODO: Support this type.
 			}
 		}
+
```

Comment:
It's better to use a standalone function to do the primary key check. `columnDefToCol` is too long.